.gitlab-ci.yml

```diff
@@ -141,7 +143,7 @@ e2e-bs:
     - yarn
     - ./scripts/ci-bs.sh test:e2e
 
-deploy-staging:
+deploy-head:
```

Comment:
what about keeping env in job names? 
ie `deploy-staging-head`, `deploy-prod-canary`, ...
it would be clearer

Reply:
Yes, done